heron/scheduler-core/src/java/com/twitter/heron/scheduler/SubmitterMain.java

```diff
@@ -315,18 +315,17 @@ public static void main(String[] args) throws Exception {
                 topologyPackage, topologyBinaryFile, topologyDefnFile, topology))
             .build());
 
-    LOG.fine("Static config loaded successfully ");
+    LOG.fine("Static config loaded successfully");
     LOG.fine(config.toString());
 
     SubmitterMain submitterMain = new SubmitterMain(config, topology);
-    boolean isSuccessful = submitterMain.submitTopology();
-
-    // Log the result and exit
-    if (!isSuccessful) {
+    try {
+      submitterMain.submitTopology();
+    } catch (TopologySubmissionException e) {
+      System.out.println(e.getMessage());
```

Comment:
If at the highest level we always want to log errors to stdout and throw a runtime exception, we should make a helper that does both of those. I know it's only two lines, but this will centralize the expected handling into a single place which will make refactoring easier and assure consistent handling.